src/groupeddataframe/grouping.jl

```diff
@@ -443,71 +612,140 @@ colwise(fns::Union{AbstractVector, Tuple}, d::AbstractDataFrame) = [f(d[i]) for
 colwise(f, gd::GroupedDataFrame) = [colwise(f, g) for g in gd]
 
 """
-Split-apply-combine in one step; apply `f` to each grouping in `d`
-based on columns `col`
+    by(d::AbstractDataFrame, keys, cols => f...; sort::Bool = false)
```

Comment:
do you want to add this drop missing keyword argument that is present in `groupby` in a separate PR?

Reply:
I'd rather add it later, this PR is already messy enough. :-)

Reply:
OK